src/common/mat_logger.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file mat_logger.h
+ * @brief Declaration file for the MatLogger class which records runtime
+ * zero-forcing matrices into csv files. Enabled or disabled by cmake.
+ */
+
+#ifndef MAT_LOGGER_H_
+#define MAT_LOGGER_H_
+
+#include "csv_logger.h"
+#include "armadillo"
+
+namespace CsvLog {
+
+constexpr size_t kFrames = 1000;
+constexpr size_t kSCs    = 304;
+constexpr size_t kBSAnts = 8;
+constexpr size_t kUEAnts = 1;
+
+#if defined(ENABLE_MAT_LOG)
+using MatBuffer = std::array<std::array<std::array<std::array<arma::cx_float,
+                  kUEAnts>, kBSAnts>, kSCs>, kFrames>;
+class MatLogger : public CsvLogger {
```

Comment:
Does it make sense for this to be inheritance or just have the logger as a member of MatLogger?  The CsvLogger class is pretty trival.